scrapy/pipelines/images.py

```diff
@@ -128,15 +131,33 @@ def get_images(self, response, request, info, *, item=None):
             raise ImageException("Image too small (%dx%d < %dx%d)" %
                                  (width, height, self.min_width, self.min_height))
 
-        image, buf = self.convert_image(orig_image)
+        if self._deprecated_convert_image is None:
+            self._deprecated_convert_image = 'response_body' not in get_func_args(self.convert_image)
+            if self._deprecated_convert_image:
+                warnings.warn('ImagesPipeline.convert_image() method overriden in a deprecated way, '
+                              'overriden method does not accept response_body argument.',
+                              category=ScrapyDeprecationWarning, stacklevel=1)
```

Comment:
:lipstick: `1` is the default stack level.

```suggestion
                              category=ScrapyDeprecationWarning)
```